CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for the Mapbox Gestures for Android
 
+## 0.8.0 - 
+#### Breaking changes
```

Comment:
also update readme for the repo with instructions how to setup Maps SDK v10?

Reply:
Added link to the https://docs.mapbox.com/android/maps/guides/user-interaction/ at https://github.com/mapbox/mapbox-gestures-android/commit/49e74da993e2324257557379a002c381a5e9bfa4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R17